.github/workflows/integration.yml

```diff
@@ -49,6 +49,9 @@ jobs:
     - uses: actions/setup-python@v5
       with:
         python-version: '3.11'
+    - uses: cvmfs-contrib/github-action-cvmfs@v4
+    - name: Test CernVM-FS
+      run: ls /cvmfs/dirac.egi.eu
```

Comment:
Is this needed? I guess it was only for testing

Reply:
Removed.